grammars/php.cson

```diff
@@ -1189,10 +1189,18 @@
         'name': 'keyword.operator.error-control.php'
       }
       {
+        'match': '(===|==|!==|!=|<>)'
+        'name': 'keyword.operator.comparison.php'
+      }
+      {
         'match': '=|\\+=|\\-=|\\*=|/=|%=|&=|\\|=|\\^=|<<=|>>='
         'name': 'keyword.operator.assignment.php'
       }
       {
+        'match': '(<=|>=|<|>)'
+        'name': 'keyword.operator.comparison.php'
```

Comment:
Because `<<=` needs to match before `<=`, but `===` needs to match before `=`.
If there's a way to do that with no two comparison matches, great.